### ISSUE TITLE
Inflation crash

### ISSUE THREAD

**Issue by AhmedML** [chrisjenx/Calligraphy#402]
The following accident sometimes occurs

```
#0. Crashed: main: 0 0 0x0000000000000000
       at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
       at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
       at com.test.core.agences.AgenceDetailFragment.onCreateView(AgenceDetailFragment.java:52)
       at android.support.v4.app.Fragment.performCreateView(Fragment.java:2184)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1298)
       at android.support.v4.app.FragmentManagerImpl.moveFragmentsToInvisible(FragmentManager.java:2323)
       at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2136)
       at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2092)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:822)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:775)
       at android.support.v4.app.FragmentActivity.onBackPressed(FragmentActivity.java:178)
       at com.test.base.BaseFragment$1.onClick(BaseFragment.java:50)
       at android.view.View.performClick(View.java:5265)
       at android.view.View$PerformClick.run(View.java:21534)
       at android.os.Handler.handleCallback(Handler.java:815)
       at android.os.Handler.dispatchMessage(Handler.java:104)
       at android.os.Looper.loop(Looper.java:207)
       at android.app.ActivityThread.main(ActivityThread.java:5728)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)

--

Fatal Exception: android.view.InflateException: Binary XML file line #95: Binary XML file line #95: Error inflating class fragment
       at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
       at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
       at com.test.agences.AgenceDetailFragment.onCreateView(AgenceDetailFragment.java:52)
       at android.support.v4.app.Fragment.performCreateView(Fragment.java:2184)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1298)
       at android.support.v4.app.FragmentManagerImpl.moveFragmentsToInvisible(FragmentManager.java:2323)
       at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2136)
       at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2092)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:822)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:775)
       at android.support.v4.app.FragmentActivity.onBackPressed(FragmentActivity.java:178)
       at com.test.base.BaseFragment$1.onClick(BaseFragment.java:50)
       at android.view.View.performClick(View.java:5265)
       at android.view.View$PerformClick.run(View.java:21534)
       at android.os.Handler.handleCallback(Handler.java:815)
       at android.os.Handler.dispatchMessage(Handler.java:104)
       at android.os.Looper.loop(Looper.java:207)
       at android.app.ActivityThread.main(ActivityThread.java:5728)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)
Caused by android.view.InflateException: Binary XML file line #95: Error inflating class fragment
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:782)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
       at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
       at com.test.core.agences.AgenceDetailFragment.onCreateView(AgenceDetailFragment.java:52)
       at android.support.v4.app.Fragment.performCreateView(Fragment.java:2184)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1298)
       at android.support.v4.app.FragmentManagerImpl.moveFragmentsToInvisible(FragmentManager.java:2323)
       at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2136)
       at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2092)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:822)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:775)
       at android.support.v4.app.FragmentActivity.onBackPressed(FragmentActivity.java:178)
       at com.test.base.BaseFragment$1.onClick(BaseFragment.java:50)
       at android.view.View.performClick(View.java:5265)
       at android.view.View$PerformClick.run(View.java:21534)
       at android.os.Handler.handleCallback(Handler.java:815)
       at android.os.Handler.dispatchMessage(Handler.java:104)
       at android.os.Looper.loop(Looper.java:207)
       at android.app.ActivityThread.main(ActivityThread.java:5728)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)
Caused by java.lang.IllegalArgumentException: Binary XML file line #95: Duplicate id 0x7f0d00cc, tag null, or parent id 0x7f0d00cb with another fragment for com.google.android.gms.maps.SupportMapFragment
       at android.support.v4.app.FragmentManagerImpl.onCreateView(FragmentManager.java:3364)
       at android.support.v4.view.LayoutInflaterCompatHC$FactoryWrapperHC.onCreateView(LayoutInflaterCompatHC.java:47)
       at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater$WrapperFactory2.onCreateView(CalligraphyLayoutInflater.java:280)
       at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:186)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:746)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
       at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
       at com.test.core.agences.AgenceDetailFragment.onCreateView(AgenceDetailFragment.java:52)
       at android.support.v4.app.Fragment.performCreateView(Fragment.java:2184)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1298)
       at android.support.v4.app.FragmentManagerImpl.moveFragmentsToInvisible(FragmentManager.java:2323)
       at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2136)
       at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2092)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:822)
       at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:775)
       at android.support.v4.app.FragmentActivity.onBackPressed(FragmentActivity.java:178)
       at com.test.base.BaseFragment$1.onClick(BaseFragment.java:50)
       at android.view.View.performClick(View.java:5265)
       at android.view.View$PerformClick.run(View.java:21534)
       at android.os.Handler.handleCallback(Handler.java:815)
       at android.os.Handler.dispatchMessage(Handler.java:104)
       at android.os.Looper.loop(Looper.java:207)
       at android.app.ActivityThread.main(ActivityThread.java:5728)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)

```

<img width="640" alt="capture d ecran 2017-07-11 a 11 23 15 a m" src="https://user-images.githubusercontent.com/24957852/28064146-6f7079cc-662b-11e7-8bbb-5ee5142d1a33.png">

**Comment by hadian90** [comment]
I'm getting the same error

**Comment by slartus** [comment]
+1

PHONE MODEL | SM-G900F
-- | --
ANDROID VERSION | 6.0.1
AVAILABLE MEM SIZE | 3887833088
STACK TRACE | android.view.InflateException: Binary XML file line #12: Binary XML file line #12: Error inflating class  	at android.view.LayoutInflater.inflate(LayoutInflater.java:551) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:429) 	at ru.karaokemenu.menu.MainMenuFragment.onCreateView(MainMenuFragment.kt:54) 	at android.support.v4.app.Fragment.performCreateView(Fragment.java:2354) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419) 	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1740) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1809) 	at android.support.v4.app.BackStackRecord.executePopOps(BackStackRecord.java:857) 	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2577) 	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2367) 	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2322) 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2229) 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700) 	at android.os.Handler.handleCallback(Handler.java:739) 	at android.os.Handler.dispatchMessage(Handler.java:95) 	at android.os.Looper.loop(Looper.java:158) 	at android.app.ActivityThread.main(ActivityThread.java:7224) 	at java.lang.reflect.Method.invoke(Native Method) 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) Caused by: android.view.InflateException: Binary XML file line #12: Error inflating class  	at android.view.LayoutInflater.createView(LayoutInflater.java:657) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.createCustomViewInternal(CalligraphyLayoutInflater.java:211) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.access$000(CalligraphyLayoutInflater.java:20) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater$PrivateWrapperFactory2.onCreateView(CalligraphyLayoutInflater.java:302) 	at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:192) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:766) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716) 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:847) 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:527) 	... 20 more Caused by: java.lang.reflect.InvocationTargetException 	at java.lang.reflect.Constructor.newInstance(Native Method) 	at android.view.LayoutInflater.createView(LayoutInflater.java:631) 	... 29 more Caused by: java.lang.OutOfMemoryError: Failed to allocate a 7361292 byte allocation with 712088 free bytes and 695KB until OOM 	at dalvik.system.VMRuntime.newNonMovableArray(Native Method) 	at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method) 	at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:856) 	at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:675) 	at android.graphics.drawable.Drawable.createFromResourceStream(Drawable.java:2228) 	at android.content.res.Resources.loadDrawableForCookie(Resources.java:4211) 	at android.content.res.Resources.loadDrawable(Resources.java:4085) 	at android.content.res.Resources.getDrawable(Resources.java:2005) 	at android.content.res.Resources.getDrawable(Resources.java:1987) 	at android.content.Context.getDrawable(Context.java:464) 	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:351) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:200) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:188) 	at android.support.v7.content.res.AppCompatResources.getDrawable(AppCompatResources.java:100) 	at android.support.v7.widget.AppCompatImageHelper.loadFromAttributes(AppCompatImageHelper.java:58) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:78) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:68) 	... 31 more android.view.InflateException: Binary XML file line #12: Error inflating class  	at android.view.LayoutInflater.createView(LayoutInflater.java:657) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.createCustomViewInternal(CalligraphyLayoutInflater.java:211) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.access$000(CalligraphyLayoutInflater.java:20) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater$PrivateWrapperFactory2.onCreateView(CalligraphyLayoutInflater.java:302) 	at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:192) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:766) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716) 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:847) 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:527) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:429) 	at ru.karaokemenu.menu.MainMenuFragment.onCreateView(MainMenuFragment.kt:54) 	at android.support.v4.app.Fragment.performCreateView(Fragment.java:2354) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419) 	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1740) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1809) 	at android.support.v4.app.BackStackRecord.executePopOps(BackStackRecord.java:857) 	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2577) 	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2367) 	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2322) 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2229) 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700) 	at android.os.Handler.handleCallback(Handler.java:739) 	at android.os.Handler.dispatchMessage(Handler.java:95) 	at android.os.Looper.loop(Looper.java:158) 	at android.app.ActivityThread.main(ActivityThread.java:7224) 	at java.lang.reflect.Method.invoke(Native Method) 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) Caused by: java.lang.reflect.InvocationTargetException 	at java.lang.reflect.Constructor.newInstance(Native Method) 	at android.view.LayoutInflater.createView(LayoutInflater.java:631) 	... 29 more Caused by: java.lang.OutOfMemoryError: Failed to allocate a 7361292 byte allocation with 712088 free bytes and 695KB until OOM 	at dalvik.system.VMRuntime.newNonMovableArray(Native Method) 	at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method) 	at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:856) 	at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:675) 	at android.graphics.drawable.Drawable.createFromResourceStream(Drawable.java:2228) 	at android.content.res.Resources.loadDrawableForCookie(Resources.java:4211) 	at android.content.res.Resources.loadDrawable(Resources.java:4085) 	at android.content.res.Resources.getDrawable(Resources.java:2005) 	at android.content.res.Resources.getDrawable(Resources.java:1987) 	at android.content.Context.getDrawable(Context.java:464) 	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:351) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:200) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:188) 	at android.support.v7.content.res.AppCompatResources.getDrawable(AppCompatResources.java:100) 	at android.support.v7.widget.AppCompatImageHelper.loadFromAttributes(AppCompatImageHelper.java:58) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:78) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:68) 	... 31 more java.lang.reflect.InvocationTargetException 	at java.lang.reflect.Constructor.newInstance(Native Method) 	at android.view.LayoutInflater.createView(LayoutInflater.java:631) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.createCustomViewInternal(CalligraphyLayoutInflater.java:211) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.access$000(CalligraphyLayoutInflater.java:20) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater$PrivateWrapperFactory2.onCreateView(CalligraphyLayoutInflater.java:302) 	at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:192) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:766) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716) 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:847) 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:527) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:429) 	at ru.karaokemenu.menu.MainMenuFragment.onCreateView(MainMenuFragment.kt:54) 	at android.support.v4.app.Fragment.performCreateView(Fragment.java:2354) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419) 	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1740) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1809) 	at android.support.v4.app.BackStackRecord.executePopOps(BackStackRecord.java:857) 	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2577) 	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2367) 	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2322) 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2229) 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700) 	at android.os.Handler.handleCallback(Handler.java:739) 	at android.os.Handler.dispatchMessage(Handler.java:95) 	at android.os.Looper.loop(Looper.java:158) 	at android.app.ActivityThread.main(ActivityThread.java:7224) 	at java.lang.reflect.Method.invoke(Native Method) 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) Caused by: java.lang.OutOfMemoryError: Failed to allocate a 7361292 byte allocation with 712088 free bytes and 695KB until OOM 	at dalvik.system.VMRuntime.newNonMovableArray(Native Method) 	at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method) 	at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:856) 	at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:675) 	at android.graphics.drawable.Drawable.createFromResourceStream(Drawable.java:2228) 	at android.content.res.Resources.loadDrawableForCookie(Resources.java:4211) 	at android.content.res.Resources.loadDrawable(Resources.java:4085) 	at android.content.res.Resources.getDrawable(Resources.java:2005) 	at android.content.res.Resources.getDrawable(Resources.java:1987) 	at android.content.Context.getDrawable(Context.java:464) 	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:351) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:200) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:188) 	at android.support.v7.content.res.AppCompatResources.getDrawable(AppCompatResources.java:100) 	at android.support.v7.widget.AppCompatImageHelper.loadFromAttributes(AppCompatImageHelper.java:58) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:78) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:68) 	... 31 more java.lang.OutOfMemoryError: Failed to allocate a 7361292 byte allocation with 712088 free bytes and 695KB until OOM 	at dalvik.system.VMRuntime.newNonMovableArray(Native Method) 	at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method) 	at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:856) 	at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:675) 	at android.graphics.drawable.Drawable.createFromResourceStream(Drawable.java:2228) 	at android.content.res.Resources.loadDrawableForCookie(Resources.java:4211) 	at android.content.res.Resources.loadDrawable(Resources.java:4085) 	at android.content.res.Resources.getDrawable(Resources.java:2005) 	at android.content.res.Resources.getDrawable(Resources.java:1987) 	at android.content.Context.getDrawable(Context.java:464) 	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:351) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:200) 	at android.support.v7.widget.AppCompatDrawableManager.getDrawable(AppCompatDrawableManager.java:188) 	at android.support.v7.content.res.AppCompatResources.getDrawable(AppCompatResources.java:100) 	at android.support.v7.widget.AppCompatImageHelper.loadFromAttributes(AppCompatImageHelper.java:58) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:78) 	at android.support.v7.widget.AppCompatImageView.(AppCompatImageView.java:68) 	at java.lang.reflect.Constructor.newInstance(Native Method) 	at android.view.LayoutInflater.createView(LayoutInflater.java:631) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.createCustomViewInternal(CalligraphyLayoutInflater.java:211) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.access$000(CalligraphyLayoutInflater.java:20) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater$PrivateWrapperFactory2.onCreateView(CalligraphyLayoutInflater.java:302) 	at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:192) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:766) 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716) 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:847) 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:527) 	at uk.co.chrisjenx.calligraphy.CalligraphyLayoutInflater.inflate(CalligraphyLayoutInflater.java:60) 	at android.view.LayoutInflater.inflate(LayoutInflater.java:429) 	at ru.karaokemenu.menu.MainMenuFragment.onCreateView(MainMenuFragment.kt:54) 	at android.support.v4.app.Fragment.performCreateView(Fragment.java:2354) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419) 	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1740) 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1809) 	at android.support.v4.app.BackStackRecord.executePopOps(BackStackRecord.java:857) 	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2577) 	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2367) 	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2322) 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2229) 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700) 	at android.os.Handler.handleCallback(Handler.java:739) 	at android.os.Handler.dispatchMessage(Handler.java:95) 	at android.os.Looper.loop(Looper.java:158) 	at android.app.ActivityThread.main(ActivityThread.java:7224) 	at java.lang.reflect.Method.invoke(Native Method) 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

**Comment by chrisjenx** [comment]
Update gradle and plugin.
